README.md

```diff
@@ -20,47 +20,78 @@
 * Polygon or polyline used for spatial 'coordinates' of a timeseries variable.
 * Mimic well known text style for encoding multipolygons with holes and multilines.
 
-## Important Elements and Structure
+## Data Elements and Structure
 
-For discussion, the structure and attributes below should be what's needed. This is using a rectangular array for the data and an indexed ragged array for the polygon nodes. In CF 2 we would expect the ragged array notation to change to a variable length data field that uses more natural 2d indexing.
+For discussion, the structure and attributes below should be what's needed to store a polygon as the `timeseries_id` of a `timeSeries` featuretype. This is using a rectangular array for the time series data and a contiguous ragged array (indexed ragged array would be silly) for the polygon nodes. In CF 2 we would expect the ragged array notation to change to a variable length data field that uses more natural 2d indexing.
```

Comment:
What we're really doing is representing polygons with instance variables describing timeSeries features, not using polygon geometry as the unique time series identifier, right?